src/FlashList.tsx

```diff
@@ -427,7 +427,7 @@ class FlashList<T> extends React.PureComponent<
     return (
       <>
         <PureComponentWrapper
-          enabled={children.length > 0 || this.isEmptyList}
+          enabled={true}
```

Comment:
Instead of this change, we can add this.isListLoaded as the first OR condition in both footer and header. Again, this is something already fixed in one of the branches.

Reply:
perfect, thanks!